### ISSUE TITLE
[DOC] gRPC bad documentation examples

### ISSUE THREAD

**Issue by santos-j** [coreemu/core#780]
**Describe the bug**
There seems to be an issue with the gRPC API wrapper in the latest version of CORE. More specifically, on the [method that serializes Node objects to Protobuf](https://github.com/coreemu/core/blob/c44dc521fb11f9308da46e0b4657ed5bc68d58fe/daemon/core/api/grpc/wrappers.py#L800). 
The nodes wouldn't be correctly instantiated, so the sessions created via the gRPC API would be left hanging in the DEFINTION of CONFIGURATION states and didn't have links correctly set up. When following the CORE tutorials, I quickly faced this issue, and wasn' t able to run either the [snippets online](https://coreemu.github.io/core/grpc.html), or the available [file examples](https://github.com/coreemu/core/tree/master/package/examples/grpc).

**To Reproduce**
1. Install CORE v9.0.2 from source on an Ubuntu 20.04
2. Install protoc v3.19.0 ([as per this link](https://github.com/coreemu/core/issues/769#issuecomment-1462352380))
3. Install EMANE v.1.3.3 from source
4. Run the gRPC API CORE tutorials from the [GitHub.IO page](https://coreemu.github.io/core/grpc.html), e.g., the [Switch example](https://github.com/coreemu/core/blob/master/package/examples/grpc/switch.py).

**Expected behavior**
Being able to run the example scripts and create sessions via the gRPC API -- the Python API was working flawlessly. 

**Screenshots**
The first bug, when trying to serialize the node type to Protobuf:

![screenshot_20230403_221222](https://user-images.githubusercontent.com/8865676/229668895-26df39a5-68ee-4e6d-b052-4dfcd51b47a3.png)

Since the node type is an enum/integer, there no need for the .value method, so I dropped it and got a second error:

![screenshot_20230403_221527](https://user-images.githubusercontent.com/8865676/229669223-0b7d6229-bda5-45da-9677-75d3b72519dc.png)

Weirdly enough, the Position object type should have a [.to_proto method ](https://github.com/coreemu/core/blob/c44dc521fb11f9308da46e0b4657ed5bc68d58fe/daemon/core/api/grpc/wrappers.py#L695), but it wasn't being called. So I ... dropped the .to_proto(), et voila, everything working. I could verify this by opening the CORE GUI and being able to load the sessions created via gRPC:

![screenshot_20230403_222022](https://user-images.githubusercontent.com/8865676/229670147-0a9c6cd6-ce16-45b9-a620-ecdbf6c258e1.png)

I tested it with different Node types and positions... and everything seems to work perfectly. And without the aforementioned modifications, the sessions wouldn't be configured properly, the links between nodes wouldn't be correctly set up, and the sessions would be left in the DEFINITION or CONFIGURATION state. Whereas after my modifications, the gRPC API was able to start everything correctly and proceed to RUNTIME. 

**Desktop (please complete the following information):**
 - OS: Ubuntu Server 20.04.5 LTS (cloud image)
 - CORE Version 9.0.2
 - EMANE Version 1.3.3

**Additional context**
You can find the diff of my changes [here](https://github.com/coreemu/core/compare/master...CCI-NextG-Testbed:core:master). If the CORE team confirms this is an issue, I can submit a PR with the fixes.

**Comment by santos-j** [comment]
After testing other functions of CORE, it looks like the change in my previous comment does fix the gRPC API from Python, but then it breaks the CORE GUI. So please disregard it! 

After investigating this matter a little more, it looks like the root cause of the problem were the gRPC examples incorrectly loading certain data classes and _enums_ from the _core_pb2_ instead of the gRPC wrappers:

![screenshot_20230404_105554](https://user-images.githubusercontent.com/8865676/229831241-bedbbb7e-e5fc-4e5a-b6e8-486778330aba.png)

The [first gRPC examples in the tutorial work great](https://coreemu.github.io/core/grpc.html). But then, the following example do this instead:

![screenshot_20230404_105639](https://user-images.githubusercontent.com/8865676/229831454-de2ef90c-8aee-4ca1-b79e-c3f57e2c417c.png)

This import from NodeType and Position from the core_pb2 seem to evaluate the NodeType as an integer instead of an _enum_, and the Position as something that doesn't have the .to_proto method defined. 

In short, gRPC API examples are not consistent, and over half of them will lead to the errors I found, due to the incorrect import of objects from the serialized protobuf object definition instead of the gRPC wrappers. 

If the maintainers confirm this is a problem, I can submit fixes for both the GitHub.IO page and file examples.

**Comment by bharnden** [comment]
Thanks for helping point out the documentation issues, I will take care of this.
